### ISSUE TITLE
Exclude / Include path handed to bup missing quotation marks

### ISSUE THREAD

**Issue by Neudrino** [spersson/Kup#32]
**Problem:**
The backup procedure fails if any directory contains white spaces in the folder name, as they are not handed over to bup command line correctly.

**Details:**
Any command line argument path handed to bup should be embraced with quotes (or otherwise handled in a proper way) to allow foldernames containing white spaces and such. Otherwise bup will not be able to process the command line argument correctly and will terminate unexpectedly.
This is a real drawback / issue, as the user should be free to choose any foldername!

**Example:**
For example PlayOnLinux Wine management program automatically creates 
*~/PlayOnLinux's virtual drives/*
which  shows up in error log as being handed over as 
*--exclude /home/$USER/PlayOnLinux's virtual drives*
and not as (note the quotes)
*--exclude "/home/$USER/PlayOnLinux's virtual drives"* .

**Expected behaviour:**
The user should be free to use any folder names he likes. Any special characters should be processed correctly. This includes white spaces (others not tested).

**Reproducibility:**
always

**Comment by Neudrino** [comment]
Hi @spersson,
I just saw that "the same" bug has been reported as #23 but I did not get the point, what is wrong.
I just checked / unchecked everything, but it is not working, to give a lame error description.
The error log suggests, that it is about quoting folder names, as I can successfully execute bup without the above mentioned folder, but inserting it the way it is showing up in error log will make bup fail...

**Comment by spersson** [comment]
Hello!

You are saying that bup "terminated unexpectedly", meaning that bup crashes when given such a folder name as an exclude. I just tried running such a command (with --exclude "test/simon's stuff") and it worked fine on version 0.27 of bup. What version  of bup do you have? (run "bup --version") Can you test running the commands that you see in the log file yourself, from a terminal? And yes... you will have to add quotation marks when you do that. :) Perhaps I should change kup to include the quotation marks when writing the log... so that it is easier to copy/paste the commands from there...
I just need some more help to find out what the real problem is... if you can experiment a bit more with running different commands, renaming stuff etc.. that could be helpful!
Anyway, thanks for reporting!

**Comment by Neudrino** [comment]
Bup version: debian/0.25-1

I just checked upon this by making a test directory with and without quotes and backing these up.
Everything worked fine there!

I thought the problem was, that you hand over the command string without correct quoting, but it is just not written in the log with correct quoting.
Adding correct quoting it will work fine on command line for me too.

The problem that Kup aboards with
``` Kup did not successfully complete the bup backup job: failed to index everything. ```
still persists.
I reviewed everything again, and it is probably no incorrect quoting issue.
The strange thing is, that I can backup some folders normal, and others giving this error. I will open a new bug report, when I get to the ground, what is hindering bup from indexing the files and folders.
